src/components/ExpandableSection/__tests__/ExpandableSection.spec.js

```diff
@@ -4,7 +4,12 @@ import { shallow } from 'enzyme';
 import { Button } from '../../../';
 import { ExpandableSection } from '../ExpandableSection';
 
-const getExpandableSection = (props = {}) => shallow(<ExpandableSection id="heyo" {...props} />);
+const child = <div className="heyo" />;
+const getExpandableSection = (props = {
+  id: 'heyo',
+  children: child,
+  title: 'Title'
+}) => shallow(<ExpandableSection {...props} />);
```

Comment:
you could also default these by `getComponent = (props = {}) => shallow(<Foo title="A" {...rest}) />`, leaving room to partially apply props via `getComponent({ children: 'bar' })`;

Reply:
Ah, I was wondering how to do that!